conf/ocsci/acm_hub_deploy_2.4.yaml

```diff
@@ -0,0 +1,5 @@
+---
+ENV_DATA:
+  deploy_acm_hub_cluster: true
+  acm_hub_channel: release-2.4
```

Comment:
is this going to be a constant one or do we need a placeholder for this too ?

Reply:
we will be creating yaml per acm release